keyboards/handwired/polly40/keymaps/default/keymap.c

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2021 PAUL ENRICO N. VIOLA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [0] = LAYOUT(
+        KC_ESC,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC, 
+        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_ENT,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_DOT,  KC_RSFT, MO(3),   
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  MO(2),   MO(1)    
+    ),
+
+    [1] = LAYOUT(
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  
+        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_ENT,  
+        KC_PSCR, KC_MPLY, KC_VOLD, KC_VOLU, KC_INS,  KC_DEL,  KC_COMM, KC_DOT,  KC_SLSH, KC_PGUP, MO(0),   
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_PGDN, KC_HOME  
+    ),
+
+    [2] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_INS,  
+        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_UP,   KC_DEL,  
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_LEFT, KC_DOWN, KC_RGHT, MO(0),   
+        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_HOME, _______  
```

Comment:
Momentary layer toggles require a transparent key beneath them, and you're only able to go upwards in the layer stack, not downwards. eg. layer 0 cannot be accessed from layer 1, like you have here.
```suggestion
        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  MO(2),   MO(1)    
    ),

    [1] = LAYOUT(
        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  
        KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_ENT,  
        KC_PSCR, KC_MPLY, KC_VOLD, KC_VOLU, KC_INS,  KC_DEL,  KC_COMM, KC_DOT,  KC_SLSH, KC_PGUP, _______,   
        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  KC_PGDN, _______  
    ),

    [2] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_INS,  
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_UP,   KC_DEL,  
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_LEFT, KC_DOWN, KC_RGHT, _______,   
        KC_LCTL, KC_LGUI, KC_LALT, KC_SPC,  _______, _______  
```